pkg/apis/stork/v1alpha1/applicationregistration.go

```diff
@@ -0,0 +1,53 @@
+package v1alpha1
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+const (
+	// ApplicationRegistrationResourceName singuler name of Application Registration CR
+	ApplicationRegistrationResourceName = "applicationregistration"
+	// ApplicationRegistrationResourcePlural plural name of Application Registration CR
+	ApplicationRegistrationResourcePlural = "applicationregistrations"
+)
+
+// +genclient
+// +genclient:nonNamespaced
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// ApplicationRegistration to collect crd resources
+type ApplicationRegistration struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+	Resources         []ApplicationResource `json:"resources"`
+}
+
+// ApplicationResource definations to collect resource and fields
+type ApplicationResource struct {
+	// ResourceKind crd kind
+	// ResourceKind string `json:"resourceKind"`
+	// CRDName to register CR on destination clusters
+	metav1.GroupVersionKind `json:",inline"`
+	// default backup/migration collects delete all resources annotation except well known,
+	// keep specific annotation
+	KeepAnnotations []string `json:"keepAnnotations"`
```

Comment:
What is the use case for this? By default, we don't delete annotations

Reply:
yes we don't, fixed